cogs/strike.py

```diff
@@ -583,7 +578,10 @@ async def _confirm_manual_add_strike(self, strike_user: discord.User | discord.M
                 await out_of_sync_ban_confirmation_message.delete()
                 return
 
-            if out_of_sync_ban_button_interaction.data["custom_id"] == "yes_out_of_sync_ban_member":  # type: ignore[index, typeddict-item] # noqa: E501
+            if (
+                out_of_sync_ban_button_interaction.data["custom_id"]  # type: ignore[index, typeddict-item]
+                == "yes_out_of_sync_ban_member"
+            ):
```

Comment:
See above comment about moving boolean expressions to separate flag rather than splitting lines